src/sql/migrations.rs

```diff
@@ -497,6 +497,18 @@ item TEXT DEFAULT '');"#,
         )
         .await?;
     }
+    if dbversion < 81 {
+        info!(context, "[migration] v81");
+        sql.execute_migration(
+            r#"CREATE TABLE w30_status_updates (
+id INTEGER PRIMARY KEY AUTOINCREMENT,
+msg_id INTEGER,
+json TEXT DEFAULT '');
```

Comment:
this is nitpicking, but i'm not a fan of calling this `json` would rather prefer `payload` or so

Reply:
i would not really care, i can change that.

Reply:
changed it, also for the sync "item", we do not use the column name "json", so that would also be in sync with that.

for the parameters, i'd still prefer `json` in the names as this also document things, however, that is also not persisted to disk and can be changed more easily.